pallets/pallet-bonded-coins/src/types.rs

```diff
@@ -9,8 +9,9 @@ pub struct Locks {
 	pub allow_swap: bool,
 }
 
-#[derive(Clone, Encode, Decode, PartialEq, Eq, TypeInfo, MaxEncodedLen, Debug)]
+#[derive(Clone, Encode, Decode, PartialEq, Eq, TypeInfo, MaxEncodedLen, Debug, Default)]
 pub enum PoolStatus<LockType> {
+	#[default]
```

Comment:
default should be locked I'd say?

Reply:
another proposal on #772 (355a49efd65cbda909183ea7f7949a3f5925b1f9)

Reply:
this conflicts with the implementation on #772 and needs to be reverted

Reply:
Ops, [5677282](https://github.com/KILTprotocol/kilt-node/pull/768/commits/5677282d8308eaa6fbb972f933ec20a3e6d30ad0)